Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM cypress/included:9.2.0
+FROM cypress/included:9.5.3
+
+# Update package lists
+apt-get install update -y
```

Comment:
RUN apt-get install update -y && apt-get curl -y

Reply:
Your wish is my command 🧞